proposals/module-linking/Explainer.md

```diff
@@ -589,15 +600,37 @@ be exported. For example:
 Therefore, module and instance types can appear in both the imports and exports
 of module types and instance types.
 
+Consequently, instances can be nested N levels deep. Correspondingly, the
+inline-alias syntax is extended to allow a sequence of N export names:
+```wasm
+(module
+  (import "i" (instance $i
+    (export "j" (instance
+      (export "k" (func))))))
+  (func (call (func $i "j" "k" "l")))
```

Comment:
Minor typo reading this again but I think the `"l"` should be dropped

Reply:
Thanks!  Fixed.